lib/client.ts

```diff
@@ -110,7 +110,47 @@ function mapError(serviceName: string, methodName: string, errResult: any) {
   throw new RpcResponseError(source, errResult);
 }
 
-class RpcResponseError {
+const EXCLUDED_META_KEYS = [
+  "type",
+  "code",
+  "expose",
+  "message",
+  "namespace",
+  "instance",
+  "source",
+];
+
+function logfmt(data: Record<string, any>) {
```

Comment:
Any reason we can't use the simpler approach `@loke/errors` uses
https://github.com/LOKE/errors/blob/f86d6143a0cbbf7370230436c96815cb208f51a5/src/util.ts#L13-L15

Reply:
It thought it didn't escape strings with spaces, but re-looking at it it just always escapes them regardless of spaces which is probably fine